apmpackage/cmd/genpackage/pipelines.go

```diff
@@ -103,7 +103,7 @@ var observerIDsPipeline = []map[string]interface{}{{
 	},
 }}
 
-var ecsVersionPipeline = []map[string]interface{}{{
+var removeECSVersionPipeline = []map[string]interface{}{{
```

Comment:
Out of curiosity, even the previous `ecsVersionPipeline` was removing the field `ecs.version`. Does this mean we were never indexing `ecs.version` so it was not used in the mappings anyway?

Reply:
The ingest pipeline was there to remove the field from documents written by older versions of apm-server. The field would get added back into documents as a `constant_keyword` field.

I've left the removal there for now, but I don't think we'll keep that in https://github.com/elastic/elasticsearch/pull/97546. We'll just dynamically map the field for older versions of apm-server - no big deal.